src/components/Search/SearchPageHeader.tsx

```diff
@@ -344,6 +344,9 @@ function SearchPageHeader({queryJSON, hash}: SearchPageHeaderProps) {
             // After https://github.com/Expensify/App/pull/51633 is merged, autocomplete functionality will be included into this component, and `getFindIDFromDisplayValue` can be removed
             const computeNodeValueFn = SearchQueryUtils.getFindIDFromDisplayValue(cardList, taxRates);
```

Comment:
```suggestion
            const computeNodeValueFn = SearchQueryUtils.getFindIDFromDisplayValue(cardList, taxRates);
            inputQueryJSON.policyID = queryJSON.policyID;
```

Reply:
This works identically, but won't break in case we add `policyID` to `SYNTAX_FILTER_KEYS`